src/Week1/Homework1.hs

```diff
@@ -0,0 +1,55 @@
+module Homework1 where
+
+-- 1
+toDigits :: Integer -> [Integer]
+toDigits num =
+  helper num []
+ where
+  helper :: Integer -> [Integer] -> [Integer]
+  helper n l =
+   if n < 1
+   then l
+   else helper (div n 10) ((mod n 10):l)
+
+toDigitsRev :: Integer -> [Integer]
+toDigitsRev num  =
+  reverse (toDigits num)
+
+-- 2
+doubleEveryOther :: [Integer] -> [Integer]
+doubleEveryOther list =
+    map altDouble isEven
+   where
+    isEven :: [(Integer, Int)]
+    isEven =
+     let len = length list
+     in zip list (reverse [1 .. len])
+
+    altDouble :: (Integer, Int) -> Integer
+    altDouble (n, i) =
+     if (mod i 2 == 0)
+     then (n * 2)
+     else n
+
+-- 3
+sumDigits :: [Integer] -> Integer
+sumDigits list = foldr (+) 0 $ concat $ map toDigits list
```

Comment:
`foldr (+) 0` is `sum` function from Prelude.
Also, `concat $ map f` can be rewritten as `concatMap f` because there's `concatMap` function in `Prelude`. And you can try to use `.` composition here as well.